<launch>
    <group ns="dodobot" >
        <node name="db_object_tracking" pkg="db_object_tracking" type="db_object_tracking.py" output="screen">
            <param name="depth_topic"           value="/camera/aligned_depth_to_color/image_raw"/>
            <param name="camera_info_topic"     value="/camera/color/camera_info"/>
            <param name="depth_info_topic"      value="/camera/aligned_depth_to_color/camera_info"/>
            <param name="detect_topic"          value="/detectnet/detections"/>
            <param name="publish_label_tfs"     value="true"/>

            <rosparam param="labels">
                - BACKGROUND
                - Apple
                - Banana
                - Grape
                - Orange
                - Pear
                - Pineapple
                - Strawberry
                - Watermelon
            </rosparam>

        </node>
    </group>
</launch>
